src/server.cpp

```diff
@@ -2063,27 +2076,32 @@ s32 Server::playSound(const SimpleSoundSpec &spec,
 		return -1;
 
 	// Create the sound
-	s32 id = m_next_sound_id++;
-	// The sound will exist as a reference in m_playing_sounds
-	m_playing_sounds[id] = ServerPlayingSound();
-	ServerPlayingSound &psound = m_playing_sounds[id];
-	psound.params = params;
-	psound.spec = spec;
+	s32 id;
+	ServerPlayingSound *psound = nullptr;
+	if (ephemeral) {
+		id = -1; // old clients will still use this, so pick a reserved ID
+	} else {
+		id = nextSoundId();
+		// The sound will exist as a reference in m_playing_sounds
+		m_playing_sounds[id] = ServerPlayingSound();
+		psound = &m_playing_sounds[id];
+		psound->params = params;
+		psound->spec = spec;
+	}
 
 	float gain = params.gain * spec.gain;
 	NetworkPacket pkt(TOCLIENT_PLAY_SOUND, 0);
 	pkt << id << spec.name << gain
 			<< (u8) params.type << pos << params.object
-			<< params.loop << params.fade << params.pitch;
+			<< params.loop << params.fade << params.pitch
+			<< ephemeral;
 
-	// Backwards compability
-	bool play_sound = gain > 0;
+	bool as_reliable = !ephemeral;
 
 	for (const u16 dst_client : dst_clients) {
-		if (play_sound || m_clients.getProtocolVersion(dst_client) >= 32) {
```

Comment:
Note: the minimum protocol version is 37 so this compatibility code can be safely removed